.github/workflows/build.yml

```diff
@@ -11,3 +11,16 @@ jobs:
         run: sudo xcode-select -switch /Applications/Xcode_13.0.app
       - name: Run pod lib lint
         run: pod lib lint
+
+  swiftlint:
+    name: SwiftLint
+    runs-on: macOS-11
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Use Xcode 13
+        run: sudo xcode-select -switch /Applications/Xcode_13.0.app     
+      - name: Install Dependencies
+        run: bundle install
+      - name: Run SwiftLint
+        run: bundle exec fastlane lint
```

Comment:
🤔  Maybe this job should go in the tests.yml instead? And have the build.yml be specific for builds per package manager?

Reply:
🤔  Yeah, it's not really a build or a test, so does feel weird having it in either of those. Looking online it looks like folks sometimes have a `.github/workflows/swiftlint.yml` but that seems too specific if we want to add other workflows that also aren't build or test specific. I'm not sure if a `run/compile.yml` or something along those lines makes sense for this type of workflow? Or maybe just doing `.github/workflows/swiftlint.yml` is fine for now and if we have too many workflows that aren't related to build or test, we could create a new workflow for those cases. That's a lot of thoughts, so curious if you/anyone else has other ideas! 😄

Reply:
A separate `swiftlint.yml` seems like a good idea to me!